spec/models/room_spec.rb

```diff
@@ -11,6 +11,8 @@
     it { is_expected.to belong_to(:user) }
     it { is_expected.to have_many(:recordings).dependent(:destroy) }
     it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_length_of(:name).is_at_least(2) }
```

Comment:
Did you try doing it in one line? `it { is_expected.to validate_length_of(:name).is_at_least(2).is_at_most(255) }

Reply:
Changes applied :heavy_check_mark: